core/domain/exp_jobs_one_off_test.py

```diff
@@ -905,9 +905,9 @@ def test_for_migration_job(self):
         }
         content2_dict = {
             'html': (
-                'Here is test case <a href="https://github.com">hello'
+                'Here is test case <a href="https://github.com">'
                 '<oppia-noninteractive-link url-with-value="&amp;quot;'
-                'here&amp;quot;" text-with-value="abc">'
+                'https://github.com&amp;quot;" text-with-value="abc">'
```

Comment:
Could you explain the rationale behind these changes?

Reply:
The cases from the dump where `a` is parent of link have the format as:
```
<a href="some url"><oppia-noninteractive-link text-with-value="some text or empty" url-with-value="same url as href">no content here</oppia-noninteractive-link></a>
```
In all these cases `a` is simply removed. The above test case follows the same format.